typescript/script/prisma/schema.prisma

```diff
@@ -1,23 +1,24 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
 datasource db {
   provider = "sqlite"
   url      = "file:./dev.db"
 }
 
-generator client {
-  provider = "prisma-client-js"
+model User {
```

Comment:
Let's use the same data model from before (so not the raw one from introspection but one that was modified after introspection like so):

```prisma
model Post {
  id        Int     @id @default(autoincrement())
  title     String
  content   String?
  published Boolean @default(false)
  author    User?   @relation(fields: [authorId], references: [id])
  authorId  Int?
}

model User {
  id    Int     @id @default(autoincrement())
  email String  @unique
  name  String?
  posts Post[]
}
```

Let's also try to retain the following order for fields:

- ID fields on top
- relation fields on the bottom
- a link field always directly follows its relation field

Reply:
Done in first example, other will follow that example.